src/SFA.DAS.AANHub.Data/Repositories/ApprenticesReadRepository.cs

```diff
@@ -17,5 +17,10 @@ internal class ApprenticesReadRepository : IApprenticesReadRepository
                 .AsNoTracking().Where(a => a.ApprenticeId == apprenticeId)
                 .SingleOrDefaultAsync();
 
+        public async Task<Apprentice?> GetApprenticeByApprenticeIdAndUserId(long apprenticeId, Guid? userId) => await _aanDataContext
+                .Apprentices
+                .AsNoTracking().Where(a => a.ApprenticeId == apprenticeId && a.MemberId == userId)
```

Comment:
This is more of a question, but what happens when you pass a valid apprenticeId but a null memberId? Is the user still found?

Reply:
`GetApprenticeByApprenticeIdAndUserId` will be deleted from the repo class and interface. Sorry about this